docs-source/spring/content/security/_index.md

```diff
@@ -2,32 +2,22 @@
 title: "Security"
 ---
 
-The Oracle Backend for Spring Boot has the following security characteristics:
+The Oracle Backend for Spring Boot and Microservices has the following security characteristics:
 
-- All access to the database is done using the Mutual Transport Layer Security (mTLS) database wallet. The user name, password, and URL are
-  stored in Kubernetes secrets.
+- All access to the database is done using the Mutual Transport Layer Security (mTLS) database wallet. The user name, password, and URL are stored in Kubernetes secrets.
 
-- Accessing the dashboards can only be done using port-forwarding.
+- Accessing the dashboards can only be done using port-forwarding. The passwords for the dashboards are provied by the installer or autogenerated and stored in k8s secrets.
 
 - The only "public" exposure is the Apache APISIX Gateway (note that the dashboard is not exposed publicly).
 
-- Services deployed to the platform are not exposed through the Apache APISIX Gateway automatically.  If you want to expose the service
-  through the gateway, you must define a route in the Apache APISIX dashboard.
+- Services deployed to the platform are not exposed through the Apache APISIX Gateway automatically.  If you want to expose the service through the gateway, you must define a route in the Apache APISIX dashboard. [APISIX Gateway Documentation](../platform/apigw)
 
 - The deployed platform services are using using self-signed certificates.
 
-- A customer application can only access services running in the platform using the Apache APISIX Gateway. The IP address for the gateway
-  can be determined by using this command:
+- A customer application can only access services running in the platform using the Apache APISIX Gateway. [APISIX Gateway Documentation](../platform/apigw)
 
-    ```bash
-    kubectl -n ingress-nginx get svc ingress-nginx-controller
-    ```
+- The customer application can be secured using Spring Security (for example, Spring OAuth 2.0 Resource Server). An external authorization server is recommended (for example Okta or Auth0).
 
-- The customer application can be secured using Spring Security (for example, Spring OAuth 2.0 Resource Server). An external authorization
-  server is recommended (for example Okta or Auth0).
+- Oracle Backend for Spring Boot and Microservices provides a AuthZ server which is an engine to authenticate and authorize requests to various components in Oracle Backend for Spring Boot. The end user can manages users using REST Endpoints. [AuthZ Server Documentation](../security/azn-server)
```

Comment:
First sentence: "... provides an AuthZ ... "
First sentence: Should "and Microservices" be added after "Spring Boot"?
Second sentence: "The end user can manage users ..."